src/Runtime/OMExternalConstant.inc

```diff
@@ -153,11 +172,11 @@ void omMMapBinaryFile(
 void omGetExternalConstantAddr(
     void **outputAddr, void **baseAddr, int64_t offset) {
   if (outputAddr == NULL) {
-    perror("Error: null pointer");
+    fprintf(stderr, "Error: null pointer.");
     return;
   }
   if (baseAddr == NULL) {
-    perror("Error: null pointer");
+    fprintf(stderr, "Error: null pointer.");
```

Comment:
No need to track these error by returning `false` or `true`?

Reply:
I can be here, but I am a bit hesitated since this is called per each constant so there would have a lot of `if then else` generated in the main inference for checking the return value of this function.